zerver/openapi/zulip.yaml

```diff
@@ -2748,7 +2748,40 @@ paths:
                                 Event sent to a user when [message flags][message-flags] are added
                                 to messages.
 
+                                This can reflect a direct user action, or can be the indirect
+                                consequence of another action. Whatever the cause, if there's a change
+                                in the set of message flags that the user has for a message, then an
+                                `update_message_flags op: add` event will be sent with the change. Note
+                                that this applies when the user already had access to the message, and
+                                continues to have access to it. When a message newly appears or
+                                disappears, a [`message`][message-event] or
+                                [`delete_message`][message-delete] event is sent instead.
+
+                                Some examples of actions that trigger an `update_message_flags op: add`
```

Comment:
I'd prefer to reserve code font for things that appear verbatim somewhere in the API, which isn't true of the string "update_message_flags op: add".

Could be written "`update_message_flags` op: `add`". Or some other wording. I don't think we have a convention we've settled on for expressing a type+op pair in the API docs.

(In chat threads I've been happy writing "an `update_message_flags`/`add` event", with a slash — and note the slash isn't in code font, because only the two pieces appear verbatim in the API, not the whole. In docs that might be a bit cryptic… though in this context it should be self-explanatory enough, and in general I guess whatever spelling we use we can linkify to make it clear.)